src/Controllers/ElementalAreaController.php

```diff
@@ -186,7 +177,7 @@ public function formAction(HTTPRequest $request)
         $formName = $request->param('FormName');
 
         // Get the element ID from the form name
-        $id = substr($formName ?? '', strlen(sprintf(self::FORM_NAME_TEMPLATE ?? '', '')));
```

Comment:
This is unrelated refactoring to remove automatically added `??` which will never be required